sci-electronics/kicad/kicad-5.1.5.ebuild

```diff
@@ -0,0 +1,131 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Can switch to EAPI=7 when wxwidgets eclass also supports it
+EAPI=6
```

Comment:
Whats holding the EAPI-7 upgrade? You can use `xdg_icon_cache_update` with EAPI-7 and drop gnome2-utils.eclass altogether. Also please leave one empty line below EAPI.